experimental/dds/tree2/src/test/feature-libraries/default-field-kinds/optionalField.spec.ts

```diff
@@ -337,7 +363,7 @@ describe("optionalField", () => {
 						[
 							fooKey,
 							[
-								{ type: Delta.MarkType.Delete, count: 1 },
+								{ type: Delta.MarkType.MoveOut, count: 1, moveId: brand(0) },
```

Comment:
I don't understand why this test expects there to be a deletion in the delta even though the field starts empty. This seems like a bug.
I'll look into it and make a separate PR to fix the code and the test expectations.

Reply:
Ah, my bad, this is for the nested change.